flask_mwoauth/__init__.py

```diff
@@ -115,14 +96,22 @@ def _prepare_long_request(url, api_query):
                 part = (k, (None, v))
             partlist.append(part)
 
-        return Request(url=url, files=partlist).prepare()
+        auth1 = OAuth1(self.consumer_token.key,
```

Comment:
I would suggest adding the parameter name for the key as well. Not exactly essential, though :-)